openmoc/thisown.i

```diff
@@ -106,6 +106,26 @@
       universes[i][j].thisown = 0
 %}
 
+/* A Lattice owns the memory for each Universe it contains */
+%pythonappend Lattice::setUniverses3D %{
+  # SWIG 3
```

Comment:
Just curious, has this been tested with both SWIG 2 and 3? If not, I have both installed and can quickly test it out.